tools/PI/DevHome.PI/BarWindow.cs

```diff
@@ -141,4 +129,25 @@ public void RotateBar()
             _horizontalWindow.Show();
         }
     }
+
+    public void UpdateBarWindowPosition(PointInt32 position)
+    {
+        _viewModel.WindowPosition = position;
+    }
+
+    public void ResetBarWindowOnTop()
+    {
+        _viewModel.IsAlwaysOnTop = true;
```

Comment:
Is this line needed?

Reply:
Yes, we need it. I added a comment explaining it.

Reply:
I wonder if this can be a method on the view model, and if the viewmodel accomplishes this by twiddling the IsAlwaysOnTop value, then so be it... the BarWindow shouldn't know how this is done.

Reply:
Sure, will update it.